src/meshapi/management/commands/scramble_members.py

```diff
@@ -19,8 +19,6 @@
 
 
 # Uses faker to get fake names, emails, and phone numbers
-# TODO: Instead of modifying real data, generate a completely fake database from
-# scratch :)
```

Comment:
@WillNilges deleted this one since it seems like we are moving forward with the "anonymized real data approach". We could also create an issue to track the task of "make a from-scratch fake data generator"